### ISSUE TITLE
EdgeDB Admin UI: Schema graph should filter by module/namespace just like the text view

### ISSUE THREAD

**Issue by CosmicPangolin** [edgedb/edgedb#7965]
Schema graph feels polluted by the now dozens of ai/auth types. I think this tool can be a valuable way of sharing app structure with non-technical team members, but right now it's information overload unless a single type is selected (so only 1 hop shown).

**Comment by haikyuu** [comment]
I agree. And while it's not straightforward in terms of UX design, I suggest exposing (or exploring) a couple of features:
- Add select box to Filter by module from User defined ones. Currently, only `schema` and `type` are there.
- Fine grained control on what appears in the "visual schema": Toggle display on/off for: a module or an object type
- The ability to store multiple "filter sessions" for easy use in the future. This can be handy when a developer is working on a specific aspect of the app.
